src/main/java/mate/academy/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -23,13 +24,34 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
             HttpHeaders headers, HttpStatusCode status,
             WebRequest request
     ) {
-        ResponseErrorDto errorDto = new ResponseErrorDto();
-        errorDto.setTime(LocalDateTime.now().toString());
-        errorDto.setStatus(HttpStatus.BAD_REQUEST.toString());
-        errorDto.setErrors(getErrors(ex));
+        ResponseErrorDto errorDto = getResponseErrorDto(
+                LocalDateTime.now(), HttpStatus.BAD_REQUEST, getErrors(ex));
         return new ResponseEntity<>(errorDto, headers, status);
     }
 
+    @ExceptionHandler(NotUniqueValueException.class)
+    public ResponseEntity<Object> handleCustomException(NotUniqueValueException ex) {
+        ResponseErrorDto errorDto = getResponseErrorDto(
+                LocalDateTime.now(), HttpStatus.BAD_REQUEST, List.of(ex.getMessage()));
+        return new ResponseEntity<>(errorDto, HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(EntityNotFoundException.class)
+    public ResponseEntity<Object> handleCustomException(EntityNotFoundException ex) {
+        ResponseErrorDto errorDto = getResponseErrorDto(
+                LocalDateTime.now(), HttpStatus.BAD_REQUEST, List.of(ex.getMessage()));
+        return new ResponseEntity<>(errorDto, HttpStatus.BAD_REQUEST);
+    }

```

Comment:
duplicate code, I think it should also work like this:
```suggestion
    @ExceptionHandler({
        EntityNotFoundException.class,
        NotUniqueValueException.class})
    public ResponseEntity<Object> handleCustomException(RuntimeException ex) {
        ResponseErrorDto errorDto = getResponseErrorDto(
                LocalDateTime.now(), HttpStatus.BAD_REQUEST, List.of(ex.getMessage()));
        return new ResponseEntity<>(errorDto, HttpStatus.BAD_REQUEST);
    }
```